spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -134,6 +135,7 @@ public Object receive(boolean expectNull) throws Exception {
 
 	@Test(expected = IllegalArgumentException.class)
 	public void testValidateKafkaTopicName() {
+		Assert.fail("Should fail now");
```

Comment:
Yes, this is to make sure Tests are triggered

Made me discover they were not o.O

Did not expect anyone to jump on this PR so quick. Still in flux, but obviously I need it to be in PR form